deploy/k8s/resume-operator.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+set -e
+
+# Scale Operator deployment back up to 1
+kubectl scale deployment nuodb-cp-operator --replicas=1
```

Comment:
Nit: Should we wait until the pod is ready?

Reply:
I don't think it's necessary. We would enable reconciliation when we want to enable readiness checks, and the 5 minute polling loops should be more than enough time for the operator to be scheduled and performing reconciliation (unless there is a long delay for it to re-acquire a lease or something).

Reply:
Make sense. The reconciliation is async anyways.